### ISSUE TITLE
Vim(let):E5555: API call: Invalid key 'zindex' Error

### ISSUE THREAD

**Issue by philippludwig** [neoclide/coc.nvim#3380]
## Result from CocInfo

```
## versions

vim version: NVIM v0.5.0-776-g6312792d8
node version: v12.22.5
coc.nvim version: 0.0.80-03c9add7cd
coc.nvim directory: /home/philipp/.vim/plugged/coc.nvim
term: tmux-256color
platform: linux

## Log of coc.nvim

2021-09-17T10:37:31.814 INFO (pid:4089090) [services] - registered service "languageserver.ccls"
2021-09-17T10:37:31.819 INFO (pid:4089090) [services] - ccls state change: stopped => starting
2021-09-17T10:37:31.835 INFO (pid:4089090) [plugin] - coc.nvim 0.0.80-03c9add7cd initialized with node: v12.22.5 after 72ms
2021-09-17T10:37:31.841 INFO (pid:4089090) [language-client-index] - Language server "languageserver.ccls" started with 4089175
2021-09-17T10:37:31.880 INFO (pid:4089090) [services] - ccls state change: starting => running
2021-09-17T10:37:31.887 INFO (pid:4089090) [services] - service languageserver.ccls started
2021-09-17T10:37:35.407 INFO (pid:4089090) [completion-complete] - Results from: file
2021-09-17T10:37:36.118 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:36.177 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:36.433 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:36.583 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:37.145 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:37.589 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:37.850 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:38.315 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:38.384 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:39.610 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:40.111 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:40.248 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:41.352 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:43.435 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:43.978 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:44.303 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:45.127 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:45.448 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:37:45.615 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:01.056 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:38:02.052 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:03.380 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:38:13.166 INFO (pid:4089090) [completion-complete] - Results from: file
2021-09-17T10:38:14.133 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:14.934 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:15.062 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:15.544 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:15.776 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:15.917 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:16.171 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:16.593 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:17.360 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:17.704 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:19.509 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:20.156 ERROR (pid:4089090) [node-client] - request error on "nvim_call_function" [
  'coc#float#create_pum_float',
  [
    0,
    0,
    [
      'This class handles the communication with the Philips IntelliVue patient monitor.'
    ],
    {
      codes: [],
      highlights: [],
      maxWidth: 80,
      pumbounding: [Object],
      focusable: 0
    }
  ]
] Vim(let):E5555: API call: Invalid key 'zindex' Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at Qy.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:13030)
    at rG.onPumChange (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:24132)
2021-09-17T10:38:20.160 ERROR (pid:4089090) [server] - unhandledRejection  Promise {
  <rejected> Error
      at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
      at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
      at Qy.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:13030)
      at rG.onPumChange (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:24132)
} Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at Qy.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:13030)
    at rG.onPumChange (/home/philipp/.vim/plugged/coc.nvim/build/index.js:238:24132)
2021-09-17T10:38:22.080 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:23.924 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:24.131 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:24.671 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:24.809 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:25.136 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:25.360 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:25.948 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:26.142 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:27.164 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:30.923 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:31.530 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:38:32.712 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:34.387 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:35.495 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:38:36.971 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:37.102 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:37.284 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:37.397 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:37.861 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:39.124 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:39.250 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:39.490 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:39.933 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:40.229 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:40.351 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:40.467 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:40.693 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:38:44.381 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:38:45.266 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:39:04.645 INFO (pid:4089090) [completion-complete] - Results from: file
2021-09-17T10:39:05.135 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:08.795 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:10.197 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:10.376 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:10.650 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:10.801 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:11.765 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:12.184 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:13.016 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:15.277 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:15.386 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:15.519 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:15.682 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:15.840 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:16.458 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:16.734 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:16.870 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:17.784 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:18.620 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:23.137 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:23.393 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:39:24.678 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:15.163 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:15.272 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:15.394 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:16.034 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:16.331 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:16.441 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:16.704 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:17.141 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:17.729 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:17.953 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:18.747 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:18.855 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:18.982 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:19.466 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:19.666 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:20.885 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:20.980 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:24.496 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:24.613 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:26.381 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:26.581 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:26.740 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:27.521 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:27.758 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:28.929 INFO (pid:4089090) [completion-complete] - Results from: file
2021-09-17T10:40:30.115 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:40:30.476 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:40:31.449 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:40:31.899 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:32.217 INFO (pid:4089090) [completion-complete] - Results from: snippets,languageserver.ccls
2021-09-17T10:40:32.769 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:32.937 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:39.470 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:39.832 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:39.969 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:41.308 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:42.188 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:42.440 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:42.794 INFO (pid:4089090) [completion-complete] - Results from: snippets
2021-09-17T10:40:46.598 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:40:47.237 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:40:47.959 INFO (pid:4089090) [completion-complete] - Results from: languageserver.ccls
2021-09-17T10:41:20.345 ERROR (pid:4089090) [node-client] - request error on "nvim_call_function" [
  'coc#float#create_cursor_float',
  [
    0,
    0,
    [ "[ccls 2] [E] use of undeclared identifier 'test'" ],
    {
      pumAlignTop: false,
      preferTop: false,
      offsetX: 0,
      title: '',
      close: 0,
      codes: [Array],
      highlights: [],
      modes: [Array],
      maxWidth: 80,
      autohide: 1
    }
  ]
] Vim(let):E5555: API call: Invalid key 'zindex' Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tr.createPopup (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:3515)
    at tr.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:2644)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async sU.echoMessage (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:15141)
    at async Timeout.<anonymous> (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:8423)
2021-09-17T10:41:20.346 ERROR (pid:4089090) [node-client] - request error nvim_call_function - Vim(let):E5555: API call: Invalid key 'zindex' Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tr.createPopup (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:3515)
    at tr.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:2644)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async sU.echoMessage (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:15141)
    at async Timeout.<anonymous> (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:8423)
2021-09-17T10:41:22.712 ERROR (pid:4089090) [node-client] - request error on "nvim_call_function" [
  'coc#float#create_cursor_float',
  [
    0,
    0,
    [ "[ccls 2] [E] use of undeclared identifier 'test'" ],
    {
      pumAlignTop: false,
      preferTop: false,
      offsetX: 0,
      title: '',
      close: 0,
      codes: [Array],
      highlights: [],
      modes: [Array],
      maxWidth: 80,
      autohide: 1
    }
  ]
] Vim(let):E5555: API call: Invalid key 'zindex' Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tr.createPopup (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:3515)
    at tr.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:2644)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async sU.echoMessage (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:15141)
    at async Timeout.<anonymous> (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:8423)
2021-09-17T10:41:22.713 ERROR (pid:4089090) [node-client] - request error nvim_call_function - Vim(let):E5555: API call: Invalid key 'zindex' Error
    at wF.request (/home/philipp/.vim/plugged/coc.nvim/build/index.js:30:34430)
    at wF.call (/home/philipp/.vim/plugged/coc.nvim/build/index.js:33:3671)
    at tr.createPopup (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:3515)
    at tr.show (/home/philipp/.vim/plugged/coc.nvim/build/index.js:146:2644)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async sU.echoMessage (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:15141)
    at async Timeout.<anonymous> (/home/philipp/.vim/plugged/coc.nvim/build/index.js:193:8423)
2021-09-17T10:41:27.823 INFO (pid:4089090) [attach] - receive notification: openLog []
2021-09-17T10:42:12.889 INFO (pid:4089090) [attach] - receive notification: updateExtensions []
2021-09-17T10:42:12.957 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:12.961 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:12.975 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:12.975 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:12.976 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:12.977 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:13.805 INFO (pid:4089090) [model-installer] - Current version 1.3.6 is up to date.
2021-09-17T10:42:13.805 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:13.806 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:13.863 INFO (pid:4089090) [model-installer] - Current version 1.1.6 is up to date.
2021-09-17T10:42:13.864 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:13.864 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:13.880 INFO (pid:4089090) [model-installer] - Current version 2.4.6 is up to date.
2021-09-17T10:42:13.881 INFO (pid:4089090) [model-installer] - Using npm from: /usr/bin/npm
2021-09-17T10:42:13.881 INFO (pid:4089090) [model-installer] - Get info from https://registry.npmjs.org/
2021-09-17T10:42:13.890 INFO (pid:4089090) [model-installer] - Current version 1.2.13 is up to date.
2021-09-17T10:42:14.016 INFO (pid:4089090) [model-installer] - Current version 1.8.6 is up to date.
2021-09-17T10:42:14.657 INFO (pid:4089090) [model-installer] - Current version 1.2.3 is up to date.
2021-09-17T10:42:14.658 WARN (pid:4089090) [plugin] - Slow action "updateExtensions" cost 1769ms
2021-09-17T10:42:39.774 INFO (pid:4089090) [attach] - receive notification: checkJsonExtension []
2021-09-17T10:42:39.830 INFO (pid:4089090) [services] - registered service "json"
2021-09-17T10:42:39.830 INFO (pid:4089090) [services] - Json language server state change: stopped => starting
2021-09-17T10:42:40.020 INFO (pid:4089090) [services] - Json language server state change: starting => running
2021-09-17T10:42:40.022 INFO (pid:4089090) [services] - service json started
2021-09-17T10:44:37.340 INFO (pid:4089090) [attach] - Request action: commandList [ 'wor', 'CocCommand wor', 14 ]
2021-09-17T10:44:38.770 INFO (pid:4089090) [attach] - receive notification: runCommand [ 'workspace.showOutput' ]
2021-09-17T10:44:42.397 WARN (pid:4089090) [plugin] - Slow action "runCommand" cost 3626ms
2021-09-17T10:44:59.891 INFO (pid:4089090) [attach] - receive notification: showInfo []
```

## Describe the bug

As of today, I get the Vim error message 
```
request error nvim_call_function - Vim(let):E5555: API call: Invalid key 'zindex' use :CocOpenLog for details
```
whenever ccls detects something that is undefined.

## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create empty folder
- Create file `mini.vim` with：

```vim
set nocompatible
set runtimepath^=/home/philipp/.vim/plugged/coc.nvim
filetype plugin indent on
syntax on
set hidden
```

- Start (neo)vim with command: `vim -u mini.vim`
- Edit a file "test.cpp": `:e test.cpp`
- Write this code:

```cpp
#include <iostream>

int main() {
test();
}
```

## Screenshots (optional)

![image](https://user-images.githubusercontent.com/3770792/133754250-f2df12c2-2e08-40f3-9025-4c5351860414.png)

## More information
- ccls version: 11.0.1-2

**Comment by fannheyward** [comment]
Same as https://github.com/neoclide/coc.nvim/commit/2fb58dc009d020bf441788ebc9dc1bd21cdb73cc#commitcomment-56403511

But I can't reproduce with nvim 0.5 https://github.com/neovim/neovim/releases/tag/v0.5.0

**Comment by philippludwig** [comment]
As you can see in the output of CocInfo, I am using 0.5.0 as well.

But it seems that the nvim versions are not unique - mine is built from commit 6312792d8a6a7d293661d33d440343d4cc6e0e6e, but the one on the release page available right now is a5ac2f45ff84a688a09479f357a9909d5b914294.

However, using the one from commit a5ac2f45ff84a688a09479f357a9909d5b914294 fixes the issue for me.

**Comment by quincyhill** [comment]
Check what version and build of neovim you are using. I had this issue until I realized I had an older 0.5.0xxx version of the neovim nightly build. I uninstalled it from the system and installed the current 0.5.1 Release build and that issue went away. Goodluck
